aten/src/ATen/FunctionalStorageImpl.cpp

```diff
@@ -10,7 +10,7 @@ namespace at::functionalization {
 
 ViewMeta ViewMeta::to_out_idx(int64_t out_idx) {
   if (out_idx == this->out_index) return *this;
-  return ViewMeta(forward_fn, reverse_fn, is_multi_output, out_idx);
+  return ViewMeta(forward_fn, reverse_fn, is_multi_output, is_as_strided, out_idx);
```

Comment:
this was such a pain to debug, bleh

Reply:
oh oops, should have caught this in code review. Next time you should add new optional parameters at the end